### PR TITLE
Fix warnings revealed by v0.14.1 PS release; export functions

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Self-explanatory. Backlinking to purescript/purescript#4071

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,10 +7,12 @@ Notable changes to this project are documented in this file. The format is based
 Breaking changes:
 
 New features:
+- Export `mapCont` and `withCont` functions (#139 by @JordanMartinez)
```
Comment (hdgarrood) [inline CHANGELOG.md:10]: If these were introduced in a separate PR and originally meant to be exported, we should probably link to the PR that actually introduced them I think.

Comment (JordanMartinez) [reply CHANGELOG.md:10]: Looks like the functions were originally added in #70

Comment (JordanMartinez) [reply CHANGELOG.md]: I've added the PR to the changelog entry. Does that suffice?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,12 @@ Notable changes to this project are documented in this file. The format is based
 Breaking changes:
 
 New features:
+- Export `mapCont` and `withCont` functions originally added in #70 by @parsonmatt (#139 by @JordanMartinez)
```
Comment (hdgarrood) [inline CHANGELOG.md:10]:
```suggestion
- Export `mapCont` and `withCont` functions originally added in #70 by @parsonsmatt (#139 by @JordanMartinez)
```

**Review by hdgarrood — APPROVED** [review]
Looks good! (apart from the typo)

**Pull request merged**